lighthouse-core/computed/metrics/first-meaningful-paint.js

```diff
@@ -26,13 +26,13 @@ class FirstMeaningfulPaint extends ComputedMetric {
    */
   static async computeObservedMetric(data) {
     const {traceOfTab} = data;
-    if (!traceOfTab.timestamps.firstMeaningfulPaint) {
+    if (traceOfTab.timings.firstMeaningfulPaint === undefined) {
```

Comment:
for these, `timestamp` is allowed to be `undefined` in the return type, so better to just check `timings` directly rather than check one and cast the other.